packages/components/Tabs/src/Tabs.windows.tsx

```diff
@@ -0,0 +1,188 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text } from '@fluentui-react-native/text';
+import { tabsName, TabsType, TabsProps, TabsState, TabsSlotProps, TabsRenderData, TabsContextData } from './Tabs.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { settings } from './Tabs.settings';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { filterViewProps } from '@fluentui-react-native/adapters';
+import { foregroundColorTokens, textTokens, backgroundColorTokens } from '@fluentui-react-native/tokens';
+import { useSelectedKey, useAsPressable } from '@fluentui-react-native/interactive-hooks';
+
+export const TabsContext = React.createContext<TabsContextData>({
+  selectedKey: null,
+  onTabsClick: (/* key: string */) => {
+    return;
+  },
+  getTabId: (/* key:string, index: number*/) => {
+    return null
+  },
+  updateSelectedTabsItemRef: (/* ref: React.RefObject<any>*/) => {
+    return;
+  },
+  tabsItemKeys: [],
+  enabledKeys: [],
```

Comment:
Two thoughts:

1. It doesn't seem like `enabledKeys` would be something that a TabItem/child of Tabs would be interested in knowing about. I guess TabsContext is starting to be used like a State here, but unless there's a reason to conflate the two I think it'd be better to keep state separate from things that child components need to know from an encapsulation standpoint.
2. To me it feels more complicated than necessary to store the enabled keys as a separate array. Since we have tabsItemKeys here, I think we could use that to determine which key to select when arrow keys are pressed by modifying the logic you already have to skip over disabled keys instead.

Reply:
1. Agree, I could move it over to state.info to keep it on the state of Tabs, but out of TabsItems and the context.

2. That is something I had tried, but for that to work, the keyboarding logic would get more complicated and Tabs would need to know which TabsItems are disabled. For Tabs to know which TabsItems are disabled, the ideas that I had were that tabsItemKeys could be made up with objects with keys and disabled status or there could be another array with disabled status. Any recommendation on how to approach this?

Also, tabsItemKeys could not be replaced by enabledKeys, as tabsItemKeys is used for accessibility.

Reply:
Ah, `tabsItemKeys` is only storing keys, right.

Since the current logic is pretty tied to having the selection tracker being a string/key instead of an index, switching to keeping the index to track the current selection instead seems like not a great idea. Given that, I'd probably lean towards having an array of tuples (`[string, boolean][]`) like you mentioned above, but the current logic also works and I think it's okay if you decide you'd rather keep it the way it is.